### ISSUE TITLE
Using Seurat object to Define Cluster Identity

### ISSUE THREAD

**Issue by spietrzak** [GreenleafLab/ArchR#267]
In the section about defining cluster identity using scRNA-seq, it says that ArchR accepts Seurat objects as input. I tried to save my Seurat object as an .rds file using the command `saveRDS(cds, file = "seRNA.rds")` where cds is the name of the Seurat object, but when I try to load the file and get the available column names for this data, I see the following output:

<img width="630" alt="Screen Shot 2020-07-30 at 11 43 45 AM" src="https://user-images.githubusercontent.com/43301299/88950308-11752180-d25a-11ea-9456-86282bf9df12.png">

Is there a particular way that I should save my Seurat object before loading it to use in ArchR? Also, in a somewhat related question, do you know how I can export the metadata/phenoData of an ArchR project object to get a table that has the cells and their cluster number, sample label, etc.?

Thanks

**Comment by jgranja24** [comment]
Hi @spietrzak,

Sorry for the delay. 

(1) phenoData isnt a function from Seurat? 

(2) You dont need to save your Seurat object prior to loading it into ArchR.

(3) If you integrate the scRNA, the nearest scRNA to each scATAC-seq cell is labeled in the ArchRProject. See the `nameCell` param in `addGeneIntegrationMatrix`. From this, you can add the other relevant info to your ArchRProject.

Best,

Jeff

**Comment by jgranja24** [comment]
Closing this issue, feel free to further comment here and open new issues!

**Comment by spietrzak** [comment]
Thank you for your response. I guess I'm still just unsure about what the best way to incorporate scRNA-seq into ArchR is. Just to explain further over what I was trying to do, I clustered scRNA-seq data in a separate RStudio Project using Seurat. I tried to save my Seurat object as an RDS file, like the example input file in the ArchR manual, and then upload this into my ArchR R Project. I'm also a little confused when you say that I don't need to save the Seurat object before loading into ArchR. If I don't save it, then how would I be able to load it into the ArchR workspace. If you have more specific steps or information about how others have done this step, that would be much appreciated.

Thank you.
